packer/configfile/configfile.go

```diff
@@ -0,0 +1,47 @@
+package configfile
+
+import (
+	"os"
+	"path/filepath"
+)
+
+const EnvPackerTmpDir = "PACKER_TMP_DIR"
+
+// ConfigFile returns the default path to the configuration file. On
+// Unix-like systems this is the ".packerconfig" file in the home directory.
+// On Windows, this is the "packer.config" file in the application data
+// directory.
+func ConfigFile() (string, error) {
```

Comment:
so the reason I moved it was because I couldn't figure out how to get rid of the include loop.
I would have preferred to leave it the top package and per comment above if I go back and define packer.TempDir() and git rid of all the 2-liners. I'll move CnfigFile() and ConfigDir() back to the top level. That was a poor choice.